lib/access.go

```diff
@@ -22,7 +22,7 @@ func (m AccessMethods) Name() string {
 // Attributes defines attributes for each method
 func (m AccessMethods) Attributes() map[string]AttributeSet {
 	return map[string]AttributeSet{
-		"createauthtoken": {AECreateAuthToken, "GET"},
+		"createauthtoken": {AECreateAuthToken, "GET", ""},
```

Comment:
with this move from 2 -> fields, and the added combination of many field values being default zero values, I think we should switch all of these to named params:

```go
"createauthtoken": { endpoint: AECreateAuthToken, verb: get }
```

I know it's not as pretty, but I think it reads easier to see each field that actually uses a non-empty-string resolver default have the field show up & change the line length